File: app/src/main/java/com/mgaetan89/showsrage/activity/EpisodeActivity.java
package com.mgaetan89.showsrage.activity;

import android.support.v4.app.Fragment;

import com.mgaetan89.showsrage.R;
import com.mgaetan89.showsrage.fragment.EpisodeFragment;

public class EpisodeActivity extends BaseActivity {
	@Override
	protected boolean displayHomeAsUp() {
		return true;
	}

	@Override
	protected Fragment getFragment() {
		return new EpisodeFragment();
	}

	@Override
	protected int getSelectedMenuId() {
		return R.id.menu_shows;
	}

	@Override
	protected int getTitleResourceId() {
		return R.string.episode;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/activity/EpisodeActivity.kt
package com.mgaetan89.showsrage.activity

import com.mgaetan89.showsrage.R
import com.mgaetan89.showsrage.fragment.EpisodeFragment

class EpisodeActivity : BaseActivity() {
    override fun displayHomeAsUp() = true

    override fun getFragment() = EpisodeFragment()

    override fun getSelectedMenuId() = R.id.menu_shows

    override fun getTitleResourceId() = R.string.episode
}
